spec/rails_helper.rb

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 require "simplecov"
-SimpleCov.start "rails"
```

Comment:
There is another call to `SimpleCov.start` in line 27. The new version of Ruby does not like that we call it more than once (see https://github.com/simplecov-ruby/simplecov/issues/1003)